_posts/2018-12-18-debugging-bad-rows-on-gcp/index.md

```diff
@@ -108,16 +111,16 @@ The above allows us to count bad rows over time - so we can see what errors we s
 > Unrecognized event [null]
 > Payload with vendor [] and version [] not supported by this version of Scala Common Enrich
 
-These aren't Snowplow events - they're mostly caused by requests hitting the Snwoplow collector, which didn't come from a Snowplow tracker.
+These aren't Snowplow events - they're mostly caused by requests which didn't come from a Snowplow tracker hitting the Snowplow collector.
```

Comment:
Let's just remove 'hitting the snowplow collector' from this sentence.

Reply:
It's syntactically confusing but also not necessary

Reply:
will do